centre-web/src/components/LaunchAppTile/List.tsx

```diff
@@ -61,12 +63,13 @@ const SortableItem = ({ item }: SortableItemProps) => {
         item={item}
         dragListeners={listeners}
         dragAttributes={attributes}
+        showDescription={showDescription}
       />
     </div>
   );
 };
 
-export const List = ({ items }: ListProps) => {
+export const List = ({ items, showDescription }: ListProps) => {
```

Comment:
if we have the useLaunchpadStore and we get the showDescription from it. Do we still have to pass it as a prop